omero/developers/Web/CreateApp.rst

```diff
@@ -6,86 +6,93 @@ to get you familiar with the Django framework. The more you know about
 Django, the easier you will find it working with the OmeroWeb framework.
 One major feature of Django that we do not use in OmeroWeb is the Django
 database mapping, since all data comes from the OMERO server and is
-saved back there. You will notice that the models.py files in each app
-are empty.
+saved back there.
 
 All official OMERO applications can be installed from PyPI_.
 
 Getting set up
 --------------
 
-In order to deploy OMERO.web in a development or testing environment please 
-follow the instructions under :doc:`Deployment`.
-
-You should make sure that you can access the webclient on
-your local machine before starting to develop your own code. Be sure to
-use the correct port number, e.g.
-
--  `http://localhost:4080/webclient/ <http://localhost:4080/webclient/>`_
-
-When you edit and save your app, Django will automatically detect this and you
-only need to refresh your browser to see the changes.
+In order to deploy OMERO.web in a development or testing environment,
+you can use Docker as described below (recommended) or 
+follow the install instructions under :doc:`Deployment`.
 
 If you want to make changes to the OMERO.web code itself, go to
 :doc:`/developers/Web/EditingOmeroWeb`.
 
-You can place your app anywhere on your :envvar:`PYTHONPATH`,
-as long as it can be imported by OMERO.web.
+Create an app from the template example
+---------------------------------------
 
-Creating an app
----------------
+Go to `https://github.com/will-moore/omero-web-apps-examples <https://github.com/will-moore/omero-web-apps-examples>`_.
```

Comment:
This url could be hidden nicely behind a reasonable text (like ``example on github`` - understood that you will be changing the link for another, but this is a different matter.